tests/smoke/setup_docker.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+set -x
+sudo sysctl -w vm.max_map_count=262144
+git clone --depth 1 https://github.com/mozilla/addons-server.git
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml pull --quiet
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml up -d --build
+# Wait for server to start
```

Comment:
It would be nice to echo this comment so that it's clear in the console that the dots represent a waiting period